src/components/table/Table.vue

```diff
@@ -295,7 +296,7 @@
              */
             isAllChecked() {
                 const validVisibleData = this.visibleData.filter(
-                        (row) => this.isRowCheckable(row) !== true)
+                        (row) => this.isRowCheckable(row) == true)
```

Comment:
Is not enough only `this.isRowCheckable(row)` ?

Reply:
How would you do it? Feel free to rewrite it in a simpler way

Reply:
I get it. I already changed it